### PR TITLE
Insert install handler into the encoding if it has been modified.

### PR DESCRIPTION
**Pull request by justMaku**

### CODE REVIEW

**Comment by DJScias** [comment]
While minor at most, I suggest fixing the tabbing of the code you inserted for installManifest.

**Comment by justMaku** [comment]
Oh, that's weird, looks fine on my machine. Computers, eh?

**Comment by justMaku** [comment]
Fixed.

File: CASCEdit/CASContainer.cs [hunk 1]
```diff
@@ -362,7 +362,11 @@ public static void Save()
 			if(InstallHandler != null)
 			{
 				Settings.Logger.LogInformation("Starting Install.");
-				InstallHandler.Write(entries);
+				var installManifest = InstallHandler.Write(entries);
+
+				if (installManifest != null) {
+					entries.Add(installManifest); //Add to entry list
```
Comment (bloerwald) [inline CASCEdit/CASContainer.cs]: Dat comment

**Comment by justMaku** [comment]
>3 spaces

Best of both worlds, eh?

**Pull request merged**